modules/test/playwright/tests/portal-workflow-kaleo-designer-web/timerNotification.spec.ts

```diff
@@ -51,4 +52,23 @@ test('LPD-16281 can create timer notifications', async ({
 	await diagramViewPage.clickReviewNodeLink();
 	
 	await nodePropertiesSidebarPage.createTimerNotification(timerNotifications);
+
+	await diagramViewPage.updateWorkflowDefinition();
+
+	await diagramViewPage.goBack();
+
+	await workflowDefinitionPage.clickSingleAproverWorkflowDefinition();
+	
+	await diagramViewPage.clickReviewNodeLink();
+
+	const timerOption = workflowDefinitionPage.page.getByRole('link', {
+		name: 'Duration: 3 week',
+	});
+
+	await expect(timerOption).toBeVisible();
+
+	await timerOption.click();
+
+	await timerPage.assertActionTimerNotification(0, timerNotifications[0]);
```

Comment:
Hmm, this does not look good... What if we make this `assertActionTimerNotifications`, pass the complete array, loop inside and user the index from the array to access what you need in `nth()`?